R/zzz.R

```diff
@@ -2,6 +2,28 @@ STORAGE_DIR <- path.expand("~/.staged.dependencies")
 STAGEDDEPS_FILENAME <- "staged_dependencies.yaml"
 CONFIG_FILENAME <- "config.yaml" # in STORAGE_DIR
 
+# Use this function when you want to change the storage directory while the package is already loaded
+# It does not delete the old storage directory
+setup_storage_dir <- function(storage_dir) {
+  STORAGE_DIR <<- storage_dir
```

Comment:
The problem with an option for storage dir is that we don't want users to change it with `options(option = new_val)` since we then do not know when it changed, so we can recreate the `config.yaml` and `package_cache` directory.